src/Parser/CMakeLists.txt

```diff
@@ -15,8 +16,13 @@ set (SRC
 
 add_library(${PROJECT_NAME}_parser ${SRC})
 
+target_link_libraries(${PROJECT_NAME}_parser
+    PUBLIC ${PROJECT_NAME}_logger
+)
+
 target_include_directories(${PROJECT_NAME}_parser
     PRIVATE ${SRC_ROOT}
+    PRIVATE ${LOG_ROOT}
```

Comment:
I believe including logger in `target_include_directories` is useless since it is automatically linked with the lib